cartoframes/data/enrichment/bigquery_client.py

```diff
@@ -0,0 +1,40 @@
+from . import fake_auth
```

Comment:
A general comment about BigQueryClient class

Could we do it more generic? I think we could decouple it from the enrichment part:
- changing dataset (`_TEMP_DATASET_ENRICHMENT`) and project (`_WORKING_PROJECT`) from const to properties received by the `upload_dataframe` & `delete_table` methods
- `upload_dataframe` should receive the schema somehow (a dict with name and type? Not sure about the options of the bigquery schema)
- moving the file to `/cartoframes/data/clients` https://github.com/CartoDB/cartoframes/tree/develop/cartoframes/data/clients

Reply:
Agree!

Reply:
Agree